modules/date-localizer/test/spec.coffee

```diff
@@ -0,0 +1,81 @@
+describe 'dateTimeLocalizerMoment', ->
+  # Not sure how to test this...
```

Comment:
This could allow passing in the moment object/function for testing purposes - then once everything is using webpack it can require('moment') and pass it in here